x/twap/types/utils.go

```diff
@@ -61,11 +61,12 @@ func AccumDiffDivDuration(accumDiff sdk.Dec, timeDelta time.Duration) sdk.Dec {
 // In other words, the first returned denom string will be the lexicographically smaller of the two denoms.
 // If the denoms are equal, an error will be returned.
 func LexicographicalOrderDenoms(denom0, denom1 string) (string, string, error) {
-	if denom0 == denom1 {
-		return "", "", fmt.Errorf("both assets cannot be of the same denom: assetA: %s, assetB: %s", denom0, denom1)
-	}
-	if denom0 > denom1 {
-		denom0, denom1 = denom1, denom0
+	if denom0 >= denom1 {
```

Comment:
I think this change only makes it more confusing by increasing the if/else branches. Do you think we can ignore it?

Reply:
Some of these really are silly, but just thought I would "fix" them all and then we could decide which of these were just simply too ridiculous to implement. Agree this one is confusing, will revert